ios/RNFetchBlobReqBuilder.m

```diff
@@ -217,36 +217,51 @@ void __block (^getFieldData)(id field) = ^(id field)
                 NSMutableData * blobData;
                 if(content != nil)
                 {
+                    void (^afterReadFile)(NSData *content, NSString * err) = ^(NSData *content, NSString * err) {
+                        if(err != nil)
+                        {
+                            onComplete(formData, YES);
+                            return;
+                        }
+                        NSString * filename = [field valueForKey:@"filename"];
+                        [formData appendData:[[NSString stringWithFormat:@"--%@\r\n", boundary] dataUsingEncoding:NSUTF8StringEncoding]];
+                        [formData appendData:[[NSString stringWithFormat:@"Content-Disposition: form-data; name=\"%@\"; filename=\"%@\"\r\n", name, filename] dataUsingEncoding:NSUTF8StringEncoding]];
+                        [formData appendData:[[NSString stringWithFormat:@"Content-Type: %@\r\n\r\n", contentType] dataUsingEncoding:NSUTF8StringEncoding]];
+                        [formData appendData:content];
+                        [formData appendData:[[NSString stringWithFormat:@"\r\n"] dataUsingEncoding:NSUTF8StringEncoding]];
+                        i++;
+                        if(i < count)
+                        {
+                            __block NSDictionary * nextField = [form objectAtIndex:i];
+                            getFieldData(nextField);
+                        }
+                        else
+                        {
+                            onComplete(formData, NO);
+                            getFieldData = nil;
+                        }
+                    };
+
+                    // Example prefix: RNFetchBlob-custom:SMCryptoFile://cmxkCg==@/Containers/Data/Application/Cache/112233.jpg
+                    NSRange matchRange = NSMakeRange(0, [content length]);
+                    NSRegularExpression* regex =
+                        [NSRegularExpression regularExpressionWithPattern:@"^RNFetchBlob-custom:([a-zA-z]+)://"
+                                                                  options:0
+                                                                    error:nil];
+                    NSArray<NSTextCheckingResult *>* matches = [regex matchesInString:content options:0 range:matchRange];
+                    if ([matches count] >= 1) {
+                        NSString *customClassName = [content substringWithRange:[matches[0] rangeAtIndex:1]];
+                        id<RNFetchBlobCustomFile> fileReader = [[NSClassFromString(customClassName) alloc] init];
```

Comment:
This line is the equivalent of the following line:

https://github.com/SymphonyOSF/react-native-wkwebview/blob/Symphony-1.10.0/ios/RCTWKWebView/RCTWKWebViewManager.m#L54

```
id<RCTWKWebViewFactory> webViewFactory = [self.bridge moduleForName:json];
```

Since there is no bridge to cross in this case, there doesn't seem to be a strong reason to use the bridge here.  Instead, the classname is directly used to get the class instance specified by the js layer

Reply:
I suggest adding some validation here:

```
Class customClass = NSClassFromString(customClassName);
if (customClass == nil) {
    onComplete(formData, YES);
    return;
}
id<RNFetchBlobCustomFile> fileReader = [[customClass alloc] init];
```